### DISCUSSION TITLE
question: Porque as discussões não estão habilitadas  no repositório?

### DISCUSSION THREAD

**Discussion by renaldodev** [angolaosc/aosc.social#24] [Ideas]
Saudações comunidade, vim para o repositório para propor algumas features que tenho interesse em trabalhar nelas, e noto que a comunidade tem discutido esses assunto na sessão de problemas(issues). Não seria melhor usar a sessão de discussão para esse propósito?

**Comment by antonio-pedro99** [comment]
Muito obrigado pela sugestão. 
A gente usa o discord para isso, mas é uma ideia legal usar as discussões

**Comment by renaldodev** [comment]
Eu entendo que o projecto ainda está pequeno mas do jeito que se está a prosseguir podemos ter alguns problemas como:

Notificações Inadequadas: Os participantes das discussões em "Issues" podem receber notificações por e-mail sempre que alguém comentar, o que pode ser inconveniente para discussões longas ou não relacionadas a problemas reais.

Impacto na Reputação: Usar a seção de "Issues" para discussões pode afetar negativamente a reputação do repositório, pois isso pode fazer com que pareça desorganizado e confuso para os contribuidores e usuários.

Dificuldade em Rastrear Discussões: À medida que o repositório cresce e acumula mais "Issues", torna-se cada vez mais difícil rastrear e encontrar discussões específicas.

**Comment by antonio-pedro99** [comment]
@renaldodev done!!
Obrigado pela sugestão
